### DISCUSSION TITLE
error: empty overlap set!

### DISCUSSION THREAD

**Discussion by Ros-i** [Saskia-Oosterbroek/decona#10] [Q&A]
Hi! First of all thanks for sharing this pipeline. We have been using it with for nanopore sequencing analysis and it worked very well.
However, in the last runs after the barcode clustering, at the [racon::Polisher::initialize] step, I got an error saying "empty overlap set!" which stopped the analysis. When I look at the files created, I see that the file that probably gives a problem is 0 KB while all the others are 1 KB. Do you maybe know what can be the problem?

**Comment by Saskia-Oosterbroek** [comment]
Hello, 
Yes this is a known issue which usually occurs with short reads. Using reads that are at least longer than 300 bases usually solves the problem for us. When you do work on a larger fragment double check if your filtering step was set correctly, the raw reads often still contain super short fragments that may cause this as well. 
I hope this works!
